example/config/experiment.yaml

```diff
@@ -299,6 +302,27 @@ grid_config:
         logical_operator: 'and'
 
 
+# PREDICTION
+# How predictions are computed for train and test matrices
+# 
+# Rank tiebreaking - In the predictions.rank_abs and rank_pct columns, ties in the score 
+# are broken either at random or based on user input. What's available for configuration
+# is very limited at the moment: a SQL clause that can go in an ORDER BY statement,
+# that has access to the predictions table.
+#
+# Example: "RANDOM()" (default),
+# Will end up in a SQL clause like: "order by score desc, RANDOM()"
+#
+# Example: "label_value desc"
```

Comment:
This is a good point. I don't really see how to make a non-confusing configuration option support joins here. Like, we could let people put in the join clause, but disconnected from the query it's probably hard to reliably write. They could put the whole query, with what's in the code file now as an example, but that is 

Honestly, given the limited scope of the usage of these columns (see my other reply just now), I'm inclined to just remove the configuration for this entirely. My opinion is that stuff like 'prioritize lowest income people in the event of ties' would belong in after-experimenting evaluation, which could be built around the enhanced `sort_predictions_and_labels` in the evaluation pull request: https://github.com/dssg/triage/pull/674/files#diff-69452711d71cc1b37559e5011b372db0 (for instance, we add onto that to make it take other arbitrary arrays that may have other data, and add some convenience onto that for getting the predictions and labels and other data)

Reply:
I think if we're reporting out best, worst, and stochastic cases in the evaluations table, it makes sense to allow those to be supported here, but I wonder if it might make more sense to just have a parameter with those explicit values allowed rather than an arbitrary `order by` clause. 

In the future, maybe that could be extended to allow specifying a join table or something along those lines for more complex cases, but may be a little safer and more explicit (especially with respect to errors confusing ascending and descending) for the current purposes.

Reply:
@shaycrk how would the stochastic case (30 trials, therefore 30 rankings) map to a ranking we can materialize here?

Reply:
Yeah, I think I got my mind all twisted thinking about this in terms of what's used for the metrics, but since there is no longer a "canonical" metric value but rather a range (worst, best) and central tendency (stochastic), there is no longer any alignment between the inserted ranks and the metric value. So I assume these will just be used for downstream processes, specifically postmodeling and bias analysis. So we should think about the configuration either in terms of what we want to support for those... or punt on configuration altogether until we start working on changes to those modules.

Reply:
Just as the random tie-breaking option (just one realization from the stochastic case) you're already implementing here. I don't think we need the flexibility of allowing the user to specify any arbitrary order by clause (which may also be error-prone) as opposed to having a fixed menu of options that mirrors what we're putting in the evaluations table.